test/e2e/framework/cmd_exec_operations.go

```diff
@@ -32,8 +32,24 @@ func NewCmdOps() CmdOps {
 }
 
 // Exec the command, exit on error
-func (co *cmdOps) Exec(command string) (stdOut, stdErr *bytes.Buffer, err error) {
+func (co *cmdOps) Exec(command string, opts ...E2EOption) (stdOut, stdErr *bytes.Buffer, err error) {
+	// Default options
+	options := &E2EOptions{
+		TanzuCommandPrefix: TanzuPrefix,
+	}
+
+	// Apply provided options
+	for _, opt := range opts {
+		opt(options)
+	}
+
+	// Verify if the tanzu prefix is set if not set it with default tanzu prefix value
+	if strings.Index(command, "%s") == 0 {
+		command = fmt.Sprintf(command, options.TanzuCommandPrefix)
+	}
```

Comment:
Currently this Exec() is used to run all command tanzu, kubectl, kind. 

Thinking through this. May be should I create a wrapper  function ExecTanzuCommand() that internally calls this Exec() and move the tanzu related options to ExecTanzuCommand() ? @anujc25 Any thoughts ?

Reply:
we cna still use the same method to run another command but if  instead of ` kubectl get po` if `%s get po` is passed it will be transformed to `tanzu get po`.

Reply:
Makes sense.